integration/kubernetes/overhead/test_fio/workloads.yaml.in

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fio
+  labels:
+    app: overhead
+spec:
+  runtimeClassName: @RUNTIMECLASS@
+  containers:
+  - name: fio
+    image: egernst/fio
```

Comment:
how much time does it take to download this image? is it too heavy? remember that part of the strategy of the CI to reduce the execution time was to choose images that did not take a lot of time to download

Reply:
the images are not so big, but I think we should have the definition here and make dockerhub generate it. I'll work on that next